src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.trim()) {
+    return {};
+  }
+
+  return sourceString
```

Comment:
The 'split' method is used correctly to separate CSS declarations, but ensure that empty declarations (e.g., consecutive semicolons) are handled properly. The current filter method on line 16 is sufficient to remove them, but this could be made more explicit in the code for readability.